src/components/explore/TranscriptContentPage.tsx

```diff
@@ -34,11 +35,11 @@ const TranscriptContentPage: FC<ITranscriptContentPage> = ({
 }) => {
   const groupedData =
     type === "alphabet"
-      ? groupDataByAlphabet(data) as GroupedData
-      : sortKeysAlphabetically(data) as GroupedData;
+      ? (groupDataByAlphabet(data) as Omit<GroupedData, "nested">)
```

Comment:
I and @0tuedon had an extensive call to refactor the grouping component. It should take a specific data type and render it. Removing the need to check if `isTranscript` or omitting `nested` field

For simplicity the data transformation logic exists outside the component and returns an array of 
``` ts
{ name: string, slug: string, count?: number, nested?: boolean }
```